external/fv3fit/tests/emulation/layers/test_normalization2.py

```diff
@@ -12,7 +12,7 @@
 
 
 def test_norm_layer():
-    norm = NormLayer(scale=2.0, center=1.0, epsilon=1.0)
+    norm = NormLayer(scale=2.0, center=1.0)
```

Comment:
I realized that none of the normalizations configurable by the microphysics emulation support `epsilon`.